SouvlakMVP/SouvlakMVP/Dijkstra.cs

```diff
@@ -6,19 +6,24 @@
 using System.Numerics;
 
 using indexT = System.Int32;
-using distanceT = System.Single;
+using edgeWeightT = System.Single;
 using System.Runtime.InteropServices;
 
 namespace SouvlakMVP;
 
-internal class Dijkstra
+public class Dijkstra
 {
-    public static (List<indexT>, distanceT) FindShortestPath(Map graph, indexT startVertex, indexT endVertex)
+    /// <summary>Finds the shortest path between a starting vertex and final vertex in a graph using Dijkstra's algorithm.</summary>
+    /// <param name="graph">The graph to search.</param>
+    /// <param name="startVertex">The vertex to start the search from.</param>
+    /// <param name="endVertex">The destination vertex of the search.</param>
+    /// <returns>A tuple containing the shortest path and the distance.</returns>
+    public static (List<indexT>, edgeWeightT) FindShortestPath(Graph graph, indexT startVertex, indexT endVertex)
```

Comment:
It lacks `early stop conditionn`:
`if currentVertex == endVertex` -> `break loop` -> `find the path`

Reply:
Done